lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -621,6 +629,13 @@ def write(chunk)
           severity = compute_severity(
             entry_resource.type, record, entry_common_labels)
 
+          # Get trace resource for LogEntry "trace" field per specified config.
```

Comment:
Let's be careful about using the word "resource" -- it means something specific in the Stackdriver context, and users may get confused. Can we use another term, like "trace id" or "span id", or am I completely confusing the terminology?